### DISCUSSION TITLE
关于算式"化简"

### DISCUSSION THREAD

**Discussion by rugdmlsy** [liusy58/NJU_AdvancedProgramming_SP22#60] [Q&A] [answered]
关于PPT上**化简要求: 成交数量一致的交易合并**需要我们实现并回显出来吗? 如果需要, 是包含在calculator函数中还是需要一个额外的函数? 会如何进行测试呢?

**Comment by RicoloveFeng** [comment, marked as answer]
不需要输出，在内部有一个处理过程就行。

合并功能不和计算器耦合，在计算器外部实现。

**Reply by PolluxyShi** [reply]
那这样的话不实现这个功能不也无伤大雅

**Reply by RicoloveFeng** [reply]
> 那这样的话不实现这个功能不也无伤大雅

见 FAQ，我们的要求重点是制造含括号的表达式。
